pkg/controllers/status.go

```diff
@@ -30,10 +31,17 @@ const (
 	defaultManagementNamespace = "openshift-cloud-controller-manager-operator"
 )
 
+type ClusterOperatorStatus struct {
```

Comment:
Thinking about better name for this struct
`ClusterOperatorStatusClient` or smth like `BaseClusterOperatorReconciler`...
Naming of this thing a bit confusing IMHO. We are using this for embedding k8s client into reconcilers.

Reply:
ClusterOperatorStatusClient works for me.